### ISSUE TITLE
Need help with workshop version

### ISSUE THREAD

**Issue by AMD-NICK** [pixeltailgames/gm-mediaplayer#18]
I don't want to use GitHub version on my server, but I want to give access for server staff to remove videos from the mediaplayer queue. How can I override `MEDIAPLAYER:IsPlayerPrivileged(ply)` method in this case? Maybe u will add hooks or something?

[My Steam discussion](http://steamcommunity.com/workshop/filedetails/discussion/546392647/1354868867727269400/)

**Comment by samuelmaddock** [comment]
Sorry, I don't follow Steam discussions as much as I should. I'll add a hook for it with the following usage:
```lua
hook.Add( "MediaPlayerIsPlayerPrivileged", "My custom permissions", function( mp, ply )
	return ply:IsAdmin()
end )
```

**Comment by samuelmaddock** [comment]
Published 7e95bbc57e4c5b3b3228e86306adc319269ec543 to workshop
